slo-monitor/src/monitors/pod_monitor.go

```diff
@@ -51,6 +53,15 @@ var (
 			Buckets: prometheus.LinearBuckets(0.5, 0.25, 50),
 		},
 	)
+
+	// PodFullStartupLatency is a prometheus metric for monitoring pod startup latency including image pull times.
+	PodFullStartupLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    PodFullStartupLatencyKey,
+			Help:    "Pod e2e startup latencies in milliseconds, with image pull times",
```

Comment:
`Pod e2e startup latencies in SECONDS, with image pull times.`?

Reply:
Oops. Thanks.